playground/css-sourcemap/__tests__/serve.spec.ts

```diff
@@ -43,19 +43,19 @@ if (!isBuild) {
     const css = await res.text()
     const map = extractSourcemap(css)
     expect(formatSourcemapForSnapshot(map)).toMatchInlineSnapshot(`
-      Object {
+      {
         "mappings": "AAAA;EACE,UAAU;AACZ;;ACAA;EACE,UAAU;AACZ",
-        "sources": Array [
+        "sources": [
           "/root/be-imported.css",
           "/root/linked-with-import.css",
         ],
-        "sourcesContent": Array [
+        "sourcesContent": [
           ".be-imported {
         color: red;
       }
       ",
           "@import '@/be-imported.css';
-
+      
```

Comment:
it looks like some extra whitespace has snuck in. also in some of the files below

Reply:
It was from Vitest. Upgraded and removed.